build.ps1

```diff
@@ -0,0 +1,18 @@
+# Include
```

Comment:
Maybe for all scripts add a small header

Reply:
All scripts have a header now and explain what they do.